1739008597317	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1739019398019	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1739019398185	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1739019398185	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1739019398186	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1739019398631	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1739019398631	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1739019398632	Sync.Service	DEBUG	User-Agent: Firefox/135.0 (Windows NT 10.0; Win64; x64) FxSync/1.137.0.20250130195129.desktop
1739019398632	Sync.Service	INFO	Starting sync at 2025-02-08 19:56:38 in browser session YuOX6QgsJwg5
1739019398632	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1739019398635	Sync.Status	INFO	Resetting Status.
1739019398635	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1739019398636	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1739019398636	FirefoxAccounts	DEBUG	getOAuthToken enter
1739019398636	FirefoxAccounts	TRACE	getCachedToken returning cached token
1739019398636	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1739019398636	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1739019398636	Sync.SyncAuthManager	INFO	Getting sync key
1739019398637	FirefoxAccounts	DEBUG	getOAuthToken enter
1739019398637	FirefoxAccounts	TRACE	getCachedToken returning cached token
1739019398637	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1739019398637	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1739019398637	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1739019398637	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1739019398637	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1739019398651	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/attached_clients
1739019398655	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2147500036 - NS_ERROR_ABORT
1739019398655	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_ABORT\"  nsresult: \"0x80004004 (NS_ERROR_ABORT)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1739019398684	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1739019398684	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1739019398685	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1739019398685	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1739019398685	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1739019398685	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1739019398686	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1739019398686	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1739019398696	Services.Common.RESTRequest	DEBUG	GET https://api.accounts.firefox.com/v1/account/attached_clients failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1739019398696	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.sys.mjs:478
1739019398696	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
request()@resource://services-common/hawkclient.sys.mjs:230
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
request()@resource://services-common/hawkclient.sys.mjs:238
1739019398697	Hawk	DEBUG	(Response) /account/attached_clients: code: null - Status text: null
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:246
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:247
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
_constructError()@resource://services-common/hawkclient.sys.mjs:113
request()@resource://services-common/hawkclient.sys.mjs:252
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:114
request()@resource://services-common/hawkclient.sys.mjs:252
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:115
request()@resource://services-common/hawkclient.sys.mjs:252
1739019398697	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_constructError()@resource://services-common/hawkclient.sys.mjs:121
request()@resource://services-common/hawkclient.sys.mjs:252
1739019398697	FirefoxAccounts	ERROR	error GETing /account/attached_clients: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1739019398697	FirefoxAccounts	ERROR	Could not update attached clients list : {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1739019398699	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/attached_clients
1739019398701	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/266395372/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1739019398701	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1739019398701	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1739019398701	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1739019398701	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1739019398702	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1739019398714	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
